common/src/main/java/com/wynntils/features/combat/ContentTrackerFeature.java

```diff
@@ -29,14 +31,28 @@ public class ContentTrackerFeature extends Feature {
 
     @SubscribeEvent
     public void onTrackerUpdate(ContentTrackerUpdatedEvent event) {
-        if (event.getName() == null) return;
-
         if (autoTrackCoordinates.get()) {
-            Models.Compass.setDynamicCompassLocation(Models.Content::getTrackedLocation);
+            Location trackedLocation = Models.Content.getTrackedLocation();
+
+            // Ideally, we would clear the compass if we do not have a
+            // tracked location. However, the event is not reliable, since
+            // it is sent multiple times with partial lore. So
+            // trackedLocation == null might also just indicate that we
+            // failed to parse, not that the quest is missing a location.
+            if (trackedLocation != null) {
+                Models.Compass.setCompassLocation(trackedLocation);
+            }
         }
 
         if (playSoundOnUpdate.get()) {
             McUtils.playSoundUI(TRACKER_UPDATE_SOUND);
         }
     }
+
+    @SubscribeEvent
+    public void onSetSpawn(SetSpawnEvent e) {
+        if (!autoTrackCoordinates.get()) return;
+
+        Models.Compass.setCompassToSpawnTracker();
```

Comment:
Why?

Reply:
Okay, I kind of see now..

Reply:
Why what?

Reply:
I didn't understand what the spawn point does in related to content tracking, but I checked now..

Reply:
Ah, maybe I should have commented on that.